plugins/qCompass/ccCompassInfo.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CC_COMPASS_INFO_HEADER
+#define CC_COMPASS_INFO_HEADER
+
+#include <qdialog.h>
+#include <qlabel.h>
+#include <qpushbutton.h>
+#include <qdialogbuttonbox.h>
+#include <qboxlayout.h>
+#include <qtextedit.h>
+
+class ccCompassInfo : public QDialog
+{
+	
+public:
+	explicit ccCompassInfo(QWidget *parent = 0) :
+		QDialog(parent)
+	{
+
+		setFixedWidth(800);
+		setFixedHeight(600);
+
+		//setup GUI components
+		QTextEdit* l = new QTextEdit(this);
+		l->acceptRichText();
+		l->setReadOnly(true);
+
+		QPushButton *okButton = new QPushButton(tr("Ok"));
+		QDialogButtonBox *buttonBox = new QDialogButtonBox(Qt::Horizontal);
+		buttonBox->addButton(okButton, QDialogButtonBox::AcceptRole);
+		connect(buttonBox, SIGNAL(accepted()), this, SLOT(accept())); //done
+		QVBoxLayout *lt = new QVBoxLayout;
+		lt->addWidget(l);
+		lt->addWidget(buttonBox);
+		setLayout(lt);
+
+		//add text
+		l->setText("<!DOCTYPE html>"
```

Comment:
You should be able to use the [Qt resource system](https://doc.qt.io/qt-5/resources.html). Essentially you add the file to a .qrc file, then to reference the HTML file you use the resource path (":foo/bar.html"). This avoids the distribution issue since it will be included in the compiled binary, but kept as HTML for editing.

(Note from the docs that you may need to use `Q_INIT_RESOURCE() `. I'm not sure quite how resources work with dynamic libs.)

If you need any help, let me know!